charts/openfga/values.schema.json

```diff
@@ -286,7 +286,22 @@
                     "type": [
                         "string",
                         "null"
-                    ]
+                    ],
+                    "description": "the URI of the datastore including credentials and database (e.g. postgres://user:password@host:port/dbname)"
+                },
+                "username": {
```

Comment:
a bit of confusion here I think, what happens if you have a username + password in the URI but also directly specified here? Would be nice to describe if one takes priority over the other.

Reply:
Now they all get set and handling is done by openfga.

So specifying username and pass would overwrite the user and pass in the URI. I'll specify this and add a reference to openfga docs, cause if that behavior changes this will also change.